src/components/pagination/_pagination.scss

```diff
@@ -115,4 +120,41 @@ $css--helpers: true;
       }
     }
   }
+
+  .bx--pagination--inline {
+    height: 42px;
+    margin-top: -0.5rem;
+    margin-bottom: -0.5rem;
+    margin-right: -1rem;
+
+    .bx--pagination__button {
+      height: 100%;
+      border-left: 1px solid $ui-04;
+      border-right: 1px solid $ui-04;
+      padding-left: 0.9rem;
```

Comment:
are we still doing no leading zeros? if so, change to `.9rem`. same applies to rest of this file

Reply:
:( prettier adds the leading zero

Reply:
ahh okay